MobileWallet/Common/MatomoTracker/MatomoTracker+SharedInstance.swift

```diff
@@ -41,25 +41,34 @@
 import Foundation
 import MatomoTracker
 
-class Tracker {
-    let matomoTracker: MatomoTracker!
+extension MatomoTracker {
```

Comment:
there's is already matomo tracking in the app, these were just events to add one so. We're using a file called Tracker.swift to abstract matomo